src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Native_x64_RustC_Rust_debuginfo2_v1.58.1.exe.sarif

```diff
@@ -951,7 +951,7 @@
             },
             {
               "id": "BA2026",
-              "name": "EnableAdditionalSdlSecurityChecks",
+              "name": "EnableMicrosoftCompilerSdlSwitch",
```

Comment:
>EnableMicrosoftCompilerSdlSwitch [](http://example.com/codeflow?start=23&length=32)

This is not related to the current change, fix by the way.

Reply:
Nice. Thanks for fixing this.